grl/policy_eval.py

```diff
@@ -36,11 +39,13 @@ def solve_mdp(self, mdp):
             a_t = np.zeros(mdp.n_states)
             a_t[s] = -1 # subtract V_pi(s) to right side
             b_t = 0
-            possible_next_ss = np.where(mdp.T[self.pi[s],s] != 0.)[0]
-            for next_s in possible_next_ss:
-                t = mdp.T[self.pi[s],s,next_s]
-                b_t -= t * mdp.R[self.pi[s],s,next_s] # subtract constants to left side
-                a_t[next_s] += t * mdp.gamma
+            t = mdp.T * self.pi_ground[s][:, None, None]
+            r = mdp.R * self.pi_ground[s][:, None, None]
+            for next_s in range(mdp.n_states):
+                t_n = t.sum(0)[s,next_s]
+                r_n = r.sum(0)[s,next_s]
+                a_t[next_s] += t_n * mdp.gamma # add V_pi(s') to right side
+                b_t -= t_n * r_n # subtract constants to left side
```

Comment:
Can you write the equation here as a comment?

Reply:
Do you mean something different than what is in the method docstring?

Reply:
Yeah like I want to see `A*V_pi = b` or something like that.

Reply:
Can go in the docstring if you prefer, or as a comment near these lines

Reply:
How's that?

Reply:
That's great